src/components/PrismicComponentCards/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-rainbow-components/components/Card';
+import './styles.css';
+
+function handleOnError(event) {
+    const { receivedEvent } = event.target;
+    receivedEvent.src = './../../../public/assets/images/ComponentDefaultImage';
```

Comment:
This can't work because you are trying to access the event.target.receivedEvent that does not exists so the later assign to receivedEvent.src fail. On the other hand if it exists then you are mutating again.
This is not the way to avoid mutation.